src/app/screens/CreatePassword.tsx

```diff
@@ -23,21 +22,15 @@ import { createPassword, useDispatch } from '@store';
 import { translate, translateRaw } from '@translations';
 import { PasswordStrength } from '@types';
 
-const CREATE_PASSWORD_STRUCT = object({
-  password: refine(string(), 'Strong password', (value) => {
-    if (passwordStrength(value).id === PasswordStrength.STRONG) {
-      return true;
-    }
-
-    return translateRaw('PASSWORD_TOO_WEAK');
-  }),
-  passwordConfirmation: refine(string(), 'Equal to password', (value, context) => {
-    if (value === context.branch[0].password) {
-      return true;
-    }
-
-    return translateRaw('PASSWORDS_NOT_EQUAL');
-  })
+const CREATE_PASSWORD_SCHEMA = object({
```

Comment:
I renamed the others to `SCHEMA` because the fact that it is for creating a password is implicit from the context. Not a necessary change, just a thought.